Scripts/Easings/PingPong.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEngine;
+
+namespace DUCK.Tween.Easings
+{
+	public static partial class Ease
+	{
+		public static class PingPong
+		{
+			private const float PI2 = Mathf.PI * 2f;
+
+			public static float Linear(float x)
+			{
+				return 1f - Mathf.Abs(2f * x - 1f);
+			}
+
+			public static float Parabola(float x)
+			{
+				var sqrRoot = 2f * x - 1f;
+				return 1f - sqrRoot * sqrRoot;
+			}
+
+			public static float Sine(float x)
+			{
+				return (Mathf.Cos(x * PI2 + Mathf.PI) + 1f) * 0.5f;
```

Comment:
I call it "Sine" but actually using Cosine -- what a troll! :trollface:

Reply:
either `y = (sin(x * 2 * pi - 0.5 * pi) + 1) * 0.5`
or `y = (cos(x * 2 * pi + pi) + 1) * 0.5`
will do the job.

I just wanted to troll. :trollface: